packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -223,6 +227,7 @@ export class InputManager {
         } else {
             pointerInfo = new PointerInfo(type, evt, null, this);
         }
+        this._movePointerInfo = pointerInfo;

```

Comment:
Should this be inside the else case above?

Reply:
You're right. I was thinking that it should get populated, regardless of direct picking but if we've already picked, we don't need to cache the pointerinfo